webdriver/webapp_server.go

```diff
@@ -41,7 +40,7 @@ type remoteAppServer struct {
 }
 
 func (i *remoteAppServer) GetWebappURL(path string) string {
-	return fmt.Sprintf("http://%s%s", i.host, path)
+	return fmt.Sprintf("https://%s%s", i.host, path)
```

Comment:
I didn't see anything that changed encryption settings for `webapp` in this PR. Will this work if `i.host` is `localhost`? Usually dev_appserver runs unencrypted.

Reply:
This is for `remoteAppServer` only, which in foreseeable future will only be `staging.wpt.fyi`, which is encrypted.

There's another `GetWebappURL` for local `DevAppServerInstance` that uses `http`.